kyuubi-common/src/main/scala/org/apache/kyuubi/events/JsonEventLogger.scala

```diff
@@ -75,6 +75,15 @@ class JsonEventLogger[T <: KyuubiEvent](logName: String,
   }
 
   private def requireLogRootWritable(): Unit = {
+    // Create logRoot
+    // Here should notice, if the user has no permission, it will throw exception
+    try {
+      FileSystem.mkdirs(fs, new Path(logRoot), JSON_LOG_DIR_PERM)
```

Comment:
LogRoot probably shares across users, is it good practice that making it be created by one single user?

Reply:
got it.
thanks.